index.html

```diff
@@ -1,51 +1,306 @@
+            </ul>
+          </div>
+          <div class="bcnbutton">
+            <a class="cbutton" href="#">See Project</a>
+          </div>
+        </li>
+        <li class="card">
+          <div class="ctext">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="cp">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="clist">
+              <li class="ciconlist">html</li>
+              <li class="ciconlist">bootstrap</li>
+              <li class="ciconlist">Ruby</li>
+            </ul>
+          </div>
+          <div class="bcnbutton">
+            <a class="cbutton" href="#">See Project</a>
+          </div>
+        </li>
+        <li class="card">
+          <div class="ctext">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="cp">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="clist">
+              <li class="ciconlist">html</li>
+              <li class="ciconlist">bootstrap</li>
+              <li class="ciconlist">Ruby</li>
+            </ul>
+          </div>
+          <div class="bcnbutton">
+            <a class="cbutton" href="#">See Project</a>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <!--Section About-->
+    <section class="about">
+      <h1 class="atext">About me</h1>
+      <p class="aptext">
+        Hello I’m a software developer! I can help you build a product, feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+     
+        <a class="abutton" href="#">See Project</a>
+      
+      <div class="aline"></div>
+      <div class="acontiner">
+        <div class="askill">
+          <div class="askilltext">
+            <div class="arectangle"></div>
+            <h3 class="askilltitle">lenguages</h3>
+          </div>
+          <ul class="askillist">
+            <li class="askillitem">
+              <p class="askillitemtext">Javascript</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Ruby</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Html</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Css</p>
+            </li>
+          </ul>
+        </div>
+        <div class="askill">
+          <div class="askilltext">
+            <div class="arectangle1"></div>
+            <h3 class="askilltitle">Frameworks</h3>
+          </div>
+          <ul class="askillist">
+            <li class="askillitem">
+              <p class="askillitemtext">Bootstrap</p>
```

Comment:
- [Optional]  Please fix all typos `Language` 
- Please follow the Figma design details, the skills should be centered, but yours is left aligned.
##### Figma Design
![Screenshot from 2022-05-28 08-49-37](https://user-images.githubusercontent.com/58771507/170816224-c35f1d5d-f0f0-4646-b6be-d9867cf1fef1.png)

##### Your design 
![Screenshot from 2022-05-28 08-52-41](https://user-images.githubusercontent.com/58771507/170816261-fb61382a-6af4-4a74-9dca-56a04edfdee6.png)